server/adapters/mysql.go

```diff
@@ -255,11 +256,39 @@ func (m *MySQL) toDeleteQuery(conditions *DeleteConditions) (string, []interface
 	return strings.Join(queryConditions, conditions.JoinCondition), values
 }
 
+//Clean deletes all records in tableName table
+func (m *MySQL) Clean(tableName string) error {
+	wrappedTx, err := m.OpenTx()
+	if err != nil {
+		return err
+	}
+
+	if err := m.cleanTableInTransaction(wrappedTx, tableName); err != nil {
+		wrappedTx.Rollback()
```

Comment:
here we don't have anything to rollback, therefore we can just use `m.datasource.ExecContext(...)` instead of using OpenTx, Commit.

Reply:
removed transaction